src/Components/CreateImageWizard/steps/Review/ReviewStep.tsx

```diff
@@ -319,7 +331,7 @@ const Review = ({ snapshottingEnabled }: { snapshottingEnabled: boolean }) => {
         {/* Intentional prop drilling for simplicity - To be removed */}
         <ContentList snapshottingEnabled={snapshottingEnabled} />
       </ExpandableSection>
-      {isUsersEnabled && (
+      {isUsersEnabled && users.length > 0 && (
```

Comment:
The expandable renders empty when the "Add user" button gets clicked. This ties to https://github.com/osbuild/image-builder-frontend/pull/2716#discussion_r1916314663, option to remove the user or to ignore the user when there are no values.

Let's maybe remove the user check for now and return to it when the step is fully implemented, what do you think?

Reply:
agree thanks